sirepo/package_data/raydata-execute-analysis.sh.jinja

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# No set -eou pipefail before activating the conda env because the
+# activation scripts have unbound variables and other errors that can
+# be ignored.
+if [[ -n "{{ conda_env }}" ]]; then
```

Comment:
Now that https://github.com/radiasoft/raydata/pull/134 is merged let's enforce that everyone use conda by getting rid of this check.

Reply:
a4175ab0127a5b2207443ee18cab9d384c401eb9